HeliosLib/Makefile

```diff
@@ -94,10 +105,9 @@ TESTS=\
 
 # target files
 ifdef WASM
-TARGETS=HeliosLib.js \
-	helios.a
```

Comment:
ohhhhhhh I already had a helios.a being generated, well I'm ahead of myself then